pyproject.toml

```diff
@@ -0,0 +1,131 @@
+[project]
+name = "cosmos-synthesizer"
+
+# Dynamic version numbers from git
+dynamic = ["version"]
+
+# Tagline for Synthesizer
+description = "A package for synthetic observations of the Universe"
+
+# A longer description of Synthesizer
+readme = "README.md"
+
+# Minimum Python version
+requires-python = ">=3.8"
+
+# The licence
+license = {file = "LICENSE.md"}
+
+# Project keywords
+keywords = [
+  "galaxy",
+  "modelling",
+  "smoothed particle hydrodynamics",
+  "particles",
+  "nbody",
+  "galaxy formation",
+  "parametric",
+  "theory",
+  "sph",
+  "cosmology",
+  "galaxy evolution",
+  "survey",
+  "space telescope",
+  "SED",
+  "sed",
+  "spectral energy distribution",
+  "stellar population",
+  "synthesis",
+]
+
+# Synthesizer authors
+authors = [
+  {name = "C. S. Fournier", email = "connor.sant.17@um.edu.mt"},
+  {name = "C. C. Lovell", email = "christopher.lovell@port.ac.uk" },
+  {name = "W. J. Roper", email = "w.roper@sussex.ac.uk" },
+  {name = "L. Seeyave", email = "L.Seeyave@sussex.ac.uk"},
+  {name = "A. J. Vijayan", email = "apavi@space.dtu.dk" },
```

Comment:
Could you change that 'J' to a 'V' here and below

Reply:
😂 no idea how that slipped through

Reply:
Done